src/resolvers/git.cr

```diff
@@ -243,7 +243,34 @@ module Shards
     end
 
     private def origin_changed?
-      (@origin_url ||= capture("git ls-remote --get-url origin").strip) != git_url
+      @origin_url ||= capture("git ls-remote --get-url origin").strip
```

Comment:
IMO this should be extracted to:

```cr
def origin_url
  @origin_url ||= capture("git ls-remote --get-url origin").strip
end
```

Then, there's no need for additional `origins_equal` method (its body can be pasted in here - into `origin_changed?`).

Reply:
Yes, I think that refactor will work fine.